docs/book/src/developer/architecture/controllers/machine.md

```diff
@@ -81,6 +81,14 @@ The `spec` object **must** at least one field defined:
 
 * `providerID` - a cloud provider ID identifying the machine.
 
+#### Optional `spec` fields
+
+The `spec` object **may** define the following field:
+
+* `osType` - string value of either `Linux` or `Windows` which can be used by providers to provide OS specific behavior.  
```

Comment:
let's also add it to the example below

Reply:
the other optional status fields were not on the example so I skipped it but happy to add it

Reply:
I have removed this altogether based on the conversation today at cluster-api meeting, the desire is remove experimental retry for for v1Alpha4. Since Windows support will be in v1alpha 3 I am going to drop the retry aspect. We can revisit the OsType field if a concrete use case comes up later.